drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -1232,9 +1232,18 @@ static const struct i2c_driver_api i2c_it8xxx2_driver_api = {
 };
 
 #ifdef CONFIG_I2C_IT8XXX2_FIFO_MODE
-BUILD_ASSERT(!((DT_INST_PROP(SMB_CHANNEL_B, fifo_enable) == true) &&
-	     (DT_INST_PROP(SMB_CHANNEL_C, fifo_enable) == true)),
-	     "Channel B and C cannot support FIFO mode at the same time.");
+/*
+ * Sometimes, channel C may write wrong register to the target device.
```

Comment:
Is this a chip errata?   Does it affect all IT8xxx2 versions?

Reply:
Yes. The variant of it81xx2 will be affected. It82xx2 variant using I2C enhance driver is not affected.